lib/runAll.js

```diff
@@ -118,6 +128,16 @@ export const runAll = async (
     logger.warn(skippingHidePartiallyStaged(hasInitialCommit && stash, diff))
   }
 
+  ctx.shouldHideUnstaged = hideUnstaged
+  if (!ctx.shouldHideUnstaged && !quiet) {
+    logger.warn(skippingHideUnstaged(hasInitialCommit && stash, diff))
+  }
+
+  ctx.shouldHideUntracked = hideUntracked
+  if (!ctx.shouldHideUntracked && !quiet) {
+    logger.warn(skippingHideUntracked(hasInitialCommit && stash, diff))
+  }
+
```

Comment:
These logs are probably unnecessary because the options are `false` by default.

Reply:
Tests are currently failing because of these logs.

Reply:
I think we don't need to log these. I'm also considering dropping the existing similar ones.

Reply:
removed